ProductsManegements/Inventory.cs

```diff
@@ -0,0 +1,198 @@
+﻿using System.Diagnostics;
+using System.Reflection;
+using System.Security.Cryptography;
+using System.Text.RegularExpressions;
+using System.Xml.Linq;
+
+namespace SimpleInventoryManagementSystem.ProductsManegements
+{
+    internal class Inventory
+    {
+        internal static List<Product> products =new();
```

Comment:
mmmmh So the product list is static and internal which means that any other class in the assembly can modify this list directly. I think It would be safer to make it private and provide public or internal methods to access or modify the list.